docs/help-center/tim-features/use-ehba.mdx

```diff
@@ -0,0 +1,61 @@
+---
+sidebar_label: Use eHBA
+title: Use eHBA
+hide_title: true
+displayed_sidebar: helpcenterSidebar
+sidebar_position: 1
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+
+<div class="hero hero--primary">
+  <div class="container">
+    <h2 class="hero__title">Use eHBA</h2>
+  </div>
+</div>
+
+## Register eHBA card
+
+You can register yourself on the TIM contact list by connecting your eHBA to your user account.  
+Thereby you will be able to contact users from other organizations and be contacted by them.  
```

Comment:
be contacted by them, as long as they too have registered an eHBA card in their account.